gen/tests/test_adminrouter_tls_conf.py

```diff
@@ -0,0 +1,115 @@
+from textwrap import dedent
+from typing import List
+
+import pytest
+
+import gen
+from gen.tests.utils import make_arguments, true_false_msg, validate_error
+
+
+class TestAdminRouterTLSConfig:
+    """
+    Tests for the Admin Router TLS Config creation.
+    """
+
+    def test_default(self):
+        """
+        By default, the configuration specifies certain TLS settings.
+
+        This test is a sanity check for the configuration template logic
+        rather than a particularly useful feature test.
+        """
+        config_path = '/etc/adminrouter-tls.conf'
+        arguments = make_arguments(new_arguments={})
+        generated = gen.generate(arguments=arguments)
+        package = generated.templates['dcos-config.yaml']['package']
+        [config] = [item for item in package if item['path'] == config_path]
+
+        expected_configuration = dedent(
+            """\
+            # Ref: https://github.com/cloudflare/sslconfig/blob/master/conf
```

Comment:
Two things:

1. As per the docstring this is mainly a sanity check. Having extra logic here to remove comments from the check is probably more hassle than having to change this test whenever there is a comment change.

2. During development I accidentally put the manual test comment inside a conditional and this caught it.

tl;dr: yes, IMO